koekalenteri-backend/src/utils/metrics.ts

```diff
@@ -0,0 +1,17 @@
+import { APIGatewayEventRequestContextWithAuthorizer } from "aws-lambda";
+const {metricScope, Unit} = require("aws-embedded-metrics");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/koekalenteri/koekalenteri/pullRequest?prid=7376473)
```suggestion
const {metricScope, Unit} = require('aws-embedded-metrics');
```